data/campaigns/Liberty/scenarios/04_Unlawful_Orders.cfg

```diff
@@ -823,6 +824,24 @@
             message= _ "We... are accursed. We were the soldiers of Prince Eldred, the King’s son and betrayer... when the Prince was slain at the hands of the mage Delfador, those of us who bore witness to it were warped into these twisted forms. Now, our mortal bodies pass... into another plane of hellish existence..."
         [/message]
 
+        [if]
+            [have_location]
+                time_of_day=lawful
+            [/have_location]
+            [then]
+                [modify_unit]
+                    [filter]
+                        id=Kestrel
+                    [/filter]
+                    name= _ "General Kestrel"
```

Comment:
Does this actually do anything? Is the name even visible at all? The unit is killed right afterwards, and he doesn't say anything else after this.

Reply:
??? How did this block of code end up there, it was supposed to be split in half before when he talked and then after for the achievement. Awkward.